### DISCUSSION TITLE
e.getDataVisibility is not a function

### DISCUSSION THREAD

**Discussion by leventquales** [chartjs/chartjs-plugin-datalabels#393] [Q&A]
Hi everyone,

I am using the latest chart.js version which is 4.4.0 and chartjs-plugin-datalabels 2.2.0 on one of my .net 5 projects and i get an error when the chart loads.

e.getDataVisibility is not a function it says, i tried other versions too. also when i hover the not-shown chart, t.getActiveElements is not a function error stacks up.

is there a different use of the plugin that i cannot find or is it someting else?

var config= {
    type: 'bar',
    data: data,           
    options: {
        responsive: true,  
        scales: {
            yAxes: [{
                ticks: {
                    beginAtZero: true
                }
            }]
        }
    },
    plugins: [ChartDataLabels]
};

var julyChart = new Chart(ctx, config);

**Comment by stockiNail** [comment]
@leventquales having a look to your chart config, it seems you are not using chart.js version 4. The config seems to be closer to the version 2. Do you have any chance to create a codepen where to see the error?
